api/client/lib.py

```diff
@@ -443,6 +443,24 @@ def get_data_series(access_token, api_host, **selection):
         raise Exception(resp.text)
 
 
+# TODO: Add code to handle no query parameters call to v2/available/sources. It is hanging at the moment.
+
+
+def get_source_ranking(access_token, api_host, series):
```

Comment:
this function returns the same thing as rank_series_by_source....

Reply:
No, it does not. get_source_ranking is called inside rank_series_by_source. The first one calls the endpoint directly and returns an array of ranked sources. The latter an array of series with all information. I separated them so we can also get the ranking only directly, without the rest of the data.